lib/cn/csched_sp3.h

```diff
@@ -45,6 +48,10 @@ struct sp3_tree {
     atomic_int       spt_enabled;
     atomic_ulong     spt_ingest_alen;
     atomic_ulong     spt_ingest_wlen;
+
+    /* Dirty node list */
+    struct mutex     spt_dlist_lock;
+    struct list_head spt_dlist;;
```

Comment:
Why is the dirty list in tree struct instead of in the sp3 struct?  All other lists of nodes are in the sp3 struct.